### PR TITLE
Added check for quoted symbol to avoid parser error in case of keyword

### LINKED ISSUE #317: Parser fails when using some keywords as column names
Parsing the following statement will result in an error
```php
new Parser('ALTER TABLE `demo` ADD KEY `IDX_REPAIR` (`REPAIR`);', true);
```

Problem is the check
```php
elseif (! empty(Parser::$STATEMENT_PARSERS[$token->value]))
```
in the parse function in AlterOperation class. But as far as I understand it is allowed to use keywords as column names if you quote them.

How can we fix the problem? The easiest would be just to remove the check in the parse function. Another one would be to add an additional attribute to a token which tells if it was quoted. In case this is true, keywords are allowed as names and we skip the additional statement error checks in the parser for that token. What do you think?

### PR DESCRIPTION
**Pull request by elliot2extreme**
Fixes #317

### CODE REVIEW

**Comment by elliot2extreme** [comment]
#317

File: src/Components/AlterOperation.php [hunk 1]
```diff
@@ -356,4 +358,14 @@ private static function checkIfColumnDefinitionKeyword($tokenValue)
         // both table as well as a specific column in the table
         return in_array($tokenValue, $common_options);
     }
+
+    /**
+     * Check if token is symbol and quoted with backtick
+     * 
+     * @param type $token token to check
```
Comment (williamdes) [inline src/Components/AlterOperation.php:365]:
```suggestion
     * @param Token $token token to check
```

**Comment by williamdes** [comment]
You seem to have a misconfigured GIT `Markus Buch <markus@macbook-pro-007.intern>` by the way ;)
Good work on this issue

**Comment by elliot2extreme** [comment]
> You seem to have a misconfigured GIT `Markus Buch <markus@macbook-pro-007.intern>` by the way ;)
> Good work on this issue

Thanks for you hint, I forgot to change it.

**Comment by williamdes** [comment]
> > You seem to have a misconfigured GIT `Markus Buch <markus@macbook-pro-007.intern>` by the way ;)
> > Good work on this issue
> 
> Thanks for you hint, I forgot to change it.

Feel free to squash and force push your commits if you want before this PR is merged :)
`git rebase -i --exec 'git commit --amend --reset-author --no-edit' 550ce6968794e890cfd5b4cbf202e716d78b6a92`

**Comment by elliot2extreme** [comment]
Is there anything more I have to do for this pr? The Travis CI seems to be failing in one case because of a missing composer lock file.

**Comment by williamdes** [comment]
> Is there anything more I have to do for this pr? The Travis CI seems to be failing in one case because of a missing composer lock file. Thanks for your support.

"6398 Segmentation fault " never mind, I consider the build to be successful https://travis-ci.org/github/phpmyadmin/sql-parser/jobs/741093817#L539

**Review by devenbansod — APPROVED** [review]
LGTM :+1: :100:

**Pull request merged**